src/lib/components/Card.svelte

```diff
@@ -166,6 +166,10 @@
 
   a.card {
     margin: 0;
+
+    &:hover {
```

Comment:
Will it override disabled state?

Reply:
Oh excellent point and well spotted!!!

Modified the style to:

```
&:not(.disabled):hover {
      color: inherit;
    }
```